### PR TITLE
chore: readme update for excludeFromSearchIndex's Docsearch limitation

### PR DESCRIPTION
**Pull request by davifantasia**
Closes https://github.com/commercetools/commercetools-docs-kit/issues/829

### CODE REVIEW

File: packages/gatsby-theme-docs/README.md [hunk 1]
```diff
@@ -107,6 +107,7 @@ module.exports = {
 - `hubspotTrackingCode` (_optional_): this is HubSpot tracking code.
 
 - `excludeFromSearchIndex` (_optional_): indicates that the website should not be indexed by crawlers. This option effectively sets the `robots="noindex"` meta attribute. **Default: `true`**
+  (Note that this dosn't currently work with Algolia's Docsearch crawler)
```
Comment (TimonRey) [inline packages/gatsby-theme-docs/README.md:110]:
```suggestion
  (Note that this doesn't currently work with Algolia's Docsearch crawler)
```

File: packages/gatsby-theme-docs/README.md [hunk 2]
```diff
@@ -179,6 +180,7 @@ Supported frontmatter options are:
 - `title` (string, **required**): the title of the page. Most of the time is the same value as in the `src/data/navigation.yaml` but it can be longer if needed.
 - `beta` (boolean): to indicate if the _beta_ info message should be displayed or not.
 - `excludeFromSearchIndex` (boolean): to indicate if the page should be excluded from being indexed by crawlers. This option effectively sets the `robots="noindex"` meta attribute.
+  (Note that this dosn't currently work with Algolia's Docsearch crawler)
```
Comment (TimonRey) [inline packages/gatsby-theme-docs/README.md:183]:
```suggestion
  (Note that this doesn't currently work with Algolia's Docsearch crawler)
```

**Review by TimonRey — CHANGES_REQUESTED** [review]
Thanks. Just a typo, the rest is good.

**Comment by TimonRey** [comment]
Do you want to include a changeset as well?

**Comment by davifantasia** [comment]
> Do you want to include a changeset as well?

Done, thanks.

**Review by TimonRey — APPROVED** [review]
Thanks

**Pull request merged**